website/src/components/DataTable/DataTable.tsx

```diff
@@ -35,7 +35,7 @@ import {
 } from "@tanstack/react-table";
 import { Filter } from "lucide-react";
 import { useTranslation } from "next-i18next";
-import { ChangeEvent, ReactNode, useState } from "react";
+import { ChangeEvent, Dispatch, ReactNode, SetStateAction, useRef, useState } from "react";
```

Comment:
are all of these imports used?

Reply:
@AbdBarho Not all of them are used, I'll remove them now. My first attempt used Dispatch and SetStateAction but I changed approaches at some point